src/opers.c

```diff
@@ -1796,6 +1796,9 @@ static inline Obj CacheOper(Obj oper, UInt i)
 
     if (cache == 0) {
         len = (i < 7 ? CACHE_SIZE * (i + 2) : CACHE_SIZE * (1 + 2));
```

Comment:
What is going on here. I'm finding it hard to see why this line does anything.

Reply:
Oh, my bad, I misread the `1` in the else part as another `i`. This line is fine